recipes/stir/meta.yaml

```diff
@@ -0,0 +1,48 @@
+
+{% set name = "STIR" %}
+{% set version = "v3_0_1" %}
+{% set sha256 = "4f257fa7a4b15c9d6eddf981d80b79fc27a3ed10df4b8184d507d82a22f3596e" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ version }}.tar.gz
+  url: https://github.com/olivierverdier/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  skip: true  # [py2k or win]
+  number: 0
+
+requirements:
+  build:
+    - numpy
+    - cmake
+    - boost
+    - swig
+    - toolchain
+  run:
+    - numpy
+
+test:
+  imports:
+    - stir
+
+about:
+  home: http://stir.sourceforge.net/
+  license: GPL and LGPL
+  license_family: GPL
+  license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE.txt'
```

Comment:
`LICENSE.txt` is in the source file, so you should just use `license_file: LICENSE.txt` and delete the license file from the recipe. This way of doing it is only if the license file isn't actually in the thing you download. :)